dotnet/src/Connectors/Connectors.UnitTests/Connectors.UnitTests.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Maybe we can also keep unit/integration tests separately per each AI service as well, to keep the convention - one test project for each soluton project.
```
 - Connectors.OpenAI
 - Connectors.OpenAI.UnitTests
 - Connectors.OpenAI.IntegrationTests
 - Connectors.HuggingFace
 - Connectors.HuggingFace.UnitTests
 - Connectors.HuggingFace.IntegrationTests
```

You would just need to rename this one to `Connectors.OpenAI.UnitTests` in this PR.